Resources/Textures/_NF/Structures/Machines/computers.rsi/meta.json

```diff
@@ -146,10 +150,74 @@
           0.3
         ]
       ]
+    },
+	{
+      "name": "shipyard_science",
+      "directions": 4,
+      "delays": [
+        [
+          1.0,
+          0.3,
+          1.0,
+          0.3
+        ],
+        [
+          1.0,
+          0.3,
+          1.0,
+          0.3
+        ],
+        [
+          1.0,
+          0.3,
+          1.0,
+          0.3
+        ],
+        [
+          1.0,
+          0.3,
+          1.0,
+          0.3
+        ]
+      ]
     },
     {
+      "name": "shipyard_science",
+      "directions": 4,
+      "delays": [
+        [
+          1.0,
+          0.3,
+          1.0,
+          0.3
+        ],
+        [
+          1.0,
+          0.3,
+          1.0,
+          0.3
+        ],
+        [
+          1.0,
+          0.3,
+          1.0,
+          0.3
+        ],
+        [
+          1.0,
+          0.3,
+          1.0,
+          0.3
+        ]
+      ]
+    },
```

Comment:
This entire block is duplicated identically right above, which is causing the YAML linter to fail.

Reply:
Ah woops